exe/floe

```diff
@@ -33,6 +34,6 @@ runner_options = opts[:docker_runner_options].to_h { |opt| opt.split("=", 2) }
 
 Floe::Workflow::Runner.docker_runner = runner_klass.new(runner_options)
 
-output = workflow.run!
+workflow.run!
 
-puts output.inspect
+puts workflow.context.state["Output"].inspect
```

Comment:
Is `workflow#output` a global concept?

Reply:
I think workflow output only makes sense once the workflow has ended, but yeah we could set a `workflow#output` once the state machine has ended